en/flight_modes/README.md

```diff
@@ -60,7 +60,14 @@ Throttle Command is mapped direct to Motor Speed.
     front-back speed. When roll and pitch are all centered (inside
     deadzone) the multirotor will hold position. Yaw controls yaw rate
     as in MANUAL mode. Throttle controls climb/descent rate as in ALTCTL
-    mode. **WARNING**: Do not Land in POSCTL Mode, use Manual Mode to Land.
+    mode. 
+    **WARNING**: Prior to Landing in PosCTL Mode be sure that Landing is 

```

Comment:
1. Generally speaking we use the same name for modes as in QGC - so use Altitude instead of ALTCTL and Position mode instead of PosCTL
1. Add a line before the **WARNING**, and make the text preceded by a carat. E.g.
```
mode.

> **WARNING**: Prior ...
```
2. Should we recommend landing is PosCTL at all? (previously suggested using manual mode, but generally Land Mode would be the right thing to use ?)

Reply:
@hamishwillee 
1. Yes this would be a good Idea. I'll change this. 

2. Comming from other Products (DJI Naza for ex.) Landing in Position Control would be a natural thing to do. This is what I did and this is where my Hexa flipped. At the time I wrote the initial Warning, Landing in Position Control Mode was not possible at all.

Reply:
> Should we recommend landing is PosCTL at all?

Yes this should work well by now. But I'd be careful on an new airframe, so I think the updated description is appropriate.

Reply:
Cool. Lets integrate this then.